src/boot_O2_g3/z_std_dma.c

```diff
@@ -202,30 +209,36 @@ s32 DmaMgr_SendRequest0(void* vramStart, u32 vromStart, u32 size) {
     return 0;
 }
 
-#ifdef NON_MATCHING
-// TODO missing a useless move initializing v0, and some reorderings
-void DmaMgr_Start() {
+void DmaMgr_Start(void) {
     DmaEntry* iter;
     u32 idx;
 
-    DmaMgr_DMARomToRam((u32)_dmadataSegmentRomStart, dmadata, (u32)(_dmadataSegmentRomEnd - _dmadataSegmentRomStart));
-
-    for (iter = dmadata, idx = 0; iter->vromEnd != 0; iter++, idx++) {
-        ;
+    DmaMgr_DMARomToRam((u32)_dmadataSegmentRomStart, (u32)dmadata,
+                       (u32)_dmadataSegmentRomEnd - (u32)_dmadataSegmentRomStart);
+
+dummy_label_580802:;
+    // for (iter = dmadata, idx = 0; iter->vromEnd != 0; iter++, idx++) {
+    //     ;
+    // }
```

Comment:
```suggestion
```
Any particular reason to keep this around?

Reply:
Either way matches. I'm not sure which is preferred for something this low-level.

Reply:
I'd prefer we pick one and then delete the other, rather then leave commented code. However I'm in the same boat and don't know which is better at this low-level.

Reply:
I also think that if we keep `dummy_label_580802`, then you could shorten it down to `dummy`